lib/config.js

```diff
@@ -69,6 +69,10 @@ const loadConfig = async (argv) => {
 
     if (extension) {
       requireLoader(extension);
+    } else {
```

Comment:
this would effectively apply `esm` for any extension not registered and be the default. I'm not sure how I feel about that, because in 99% of cases it wouldn't be necessary. On line 37 you can see how `rechoir` is setup, and It'd be grand if `esm` were able to roll with that the same way. The last time I checked `esm` was still insisting upon a different approach of having to pass `module`.